spec/views/social_networking/profile_pages/show.html.erb_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+
+module SocialNetworking
+  RSpec.describe "social_networking/profile_pages/show.html.erb", type: :view do
+    fixtures :all
```

Comment:
I may be mistaken, but I believe Eric made a case for slowly phasing out the use of fixtures for unit testing and focus more on doubles. My main issues with fixtures are: they significantly slow down tests and they cause test logic to bleed throughout the test suites.